README.md

```diff
@@ -1490,6 +1490,7 @@ _Related: [Monitoring](#monitoring)_
 - [Piwigo](https://piwigo.org/) - Photo gallery software for the web, built by an active community of users and developers. ([Source Code](https://github.com/Piwigo/Piwigo)) `GPL-2.0` `PHP`
 - [Quru Image Server](https://quruimageserver.com/) - High performance dynamically resizing image server offering directory based access control cropping, rotation, color management and other tools. ([Demo](https://quruimageserver.com), [Source Code](https://github.com/quru/qis)) `AGPL-3.0` `Python`
 - [sigal](https://github.com/saimn/sigal) - Yet another simple static gallery generator. `MIT` `Python`
+- [Thumbor](http://thumbor.org/) - A smart imaging service and enables on-demand cropping, resizing, applying filters and optimizing images. ([Source Code](https://github.com/thumbor/thumbor)) `MIT` `Docker/Python`
```

Comment:
The installation docs don't mention Docker https://thumbor.readthedocs.io/en/latest/installing.html, or did I miss something? Please change the language/platform tag to just `Python`

Reply:
Since the recommended install method does not use Docker, I'd rather add it as `Python/Docker` (not the other way around)